### ISSUE TITLE
Privacy redirect is very invonsistent in what it stands behind

### ISSUE THREAD

**Issue by tio-trom** [SimonBrazell/privacy-redirect#273]
So...Privacy redirect labels websites like Youtube, Twitter, Instagram or other Google services as not privacy friendly, that's why it adds a privacy redirect for them. But then it redirects Reddit to old.reddit.com that is full of ads and trackers; it redirects search engines to DuckDuckGo, Startpage, Ecosia or Qwant which are ad-based and track your IP and more; While at the same time there is a [wild assumption that Wikipedia might be infiltrated by the NSA](https://github.com/SimonBrazell/privacy-redirect/issues/232) so we need a Wikipedia Privacy alternative!?

I do not understand...

Reddit can be redirected to teddit or libreddit and makes perfect sense, no need to use old.reddit.com or reddit mobile. Are you using the mobile version of Instagram to label it as a privacy redirect option? No.

Search engines can be easily replaced by SearX and SearX only. Tons of instances, they work great. No excuse.

Please do not deviate from your wonderful core idea of Privacy. I love this extension and we use it as default for our Linux distribution, but I think it has lost the meaning of "privacy" in the above examples.

**Comment by SimonBrazell** [comment]
Hi @tio-trom yes the waters have been a little muddied with recent user submissions to the project. 

Wikipedia redirects seemed pretty popular so I went with popular opinion on that one, I personally have no issues with Wikipedia and I think you made some pretty valid points in the other thread discussing it. 

The reddit instances could be culled to only include those that protect privacy, as the user is free to add their own instances anyway, same with the search instances but we will need to implement custom instances for search first, which is a little more involved than the others because it also includes the query string.

**Comment by Kotuklion** [comment]
I agree with everything, maybe except for wikiless, but there already has been a long [discussion](https://github.com/SimonBrazell/privacy-redirect/issues/232) about that :) It's true that we need some kind of custom instances management, so that people could freely remove and add the instances that they would like to use.
>Search engines can be easily replaced by SearX and SearX only.
What about Whoogle?

**Comment by tio-trom** [comment]
> What about Whoogle?

I am not familiar with it but if does not injects trackers/ads then is fit for this extension in my view.

**Comment by Quix0r** [comment]
How about @Searx project? It is a meta search-engine and adds no tracker anywhere.

**Comment by tio-trom** [comment]
> How about @searx project? It is a meta search-engine and adds no tracker anywhere.

That's already added. It is what I surely suggest and we also provide an instance of that at search.trom.tf . I use it daily and works great.

**Comment by futureisfoss** [comment]
[Whoogle](https://github.com/benbusby/whoogle-search) is a good alternative to https://startpage.com/. It makes the transition easier for people to switch from google, cause whoogle has a similar interface and it fetches results directly from google. [Searx](https://github.com/searx/searx) is more feature full and advanced, its what I use personally.

I find it absurd to see people using custom front-ends for Wikipedia. Wikipedia is so simple that it works very well even with JS blocked. Combine it with a proxy/vpn/tor and you solved all the privacy concerns. I explained it in more depth here - https://github.com/SimonBrazell/privacy-redirect/issues/232#issuecomment-892120527

**Comment by Jerriy** [comment]
> 
> 
> [Whoogle](https://github.com/benbusby/whoogle-search) is a good alternative to https://startpage.com/. It makes the transition easier for people to switch from google, cause whoogle has a similar interface and it fetches results directly from google. [Searx](https://github.com/searx/searx) is more feature full and advanced, its what I use personally.
> 
> I find it absurd to see people using custom front-ends for Wikipedia. Wikipedia is so simple that it works very well even with JS blocked. Combine it with a proxy/vpn/tor and you solved all the privacy concerns. I explained it in more depth here - [#232 (comment)](https://github.com/SimonBrazell/privacy-redirect/issues/232#issuecomment-892120527)

No, Wikipedia is also needed. Some of us don't have VPN access on all our PCs so if you happen to be on an IP that has been banned by Wikipedia then it sucks. With Wikiless, that problem is instantly solved without me having to tinker my browser's vpn connection.

**Comment by Jerriy** [comment]
> 
> 
> I agree with everything, maybe except for wikiless, but there already has been a long [discussion](https://github.com/SimonBrazell/privacy-redirect/issues/232) about that :) **It's true that we need some kind of custom instances management**, so that people could freely remove and add the instances that they would like to use.
> 
> > Search engines can be easily replaced by SearX and SearX only.
> 
> What about Whoogle?

Agree with the highlighted bit. We need an indication in each available alternative option whether it is privacy maximum alternative (with no tracking) or whether it's just merely an improvement on the main portal but still not pure privacy alternative (aka reddit versus old.reddit).

**Comment by tio-trom** [comment]
@Jerriy  I understand you, is just that we wonder what is this extension about? Is it acting like a proxy/vpn to circumvent censorship? Or is it more like an adblocker to stop tracking and ads? From my understanding the latest is more correct.

**Comment by futureisfoss** [comment]
> No, Wikipedia is also needed. Some of us don't have VPN access on all our PCs so if you happen to be on an IP that has been banned by Wikipedia then it sucks. With Wikiless, that problem is instantly and automatically solved... without me having to tinker my browser's vpn connection.

Why don't you use https://searx.tux.land/morty/?mortyurl=https%3A%2F%2Fen.wikipedia.org%2Fwiki%2FMain_Page&mortyhash=7ebaec8bd16aa09cda5ea1ab60a80644ab14c4ae72d9f257e665aad197b40511 for that ? Why do we need a custom frond-end specifically designed for wikipedia when we have services like morty that works on all websites. Just go to a searx instance like https://searx.tux.land and search for any website you want, and instead of clicking the main link, select the proxy button bellow it. Morty is way better than wikiless because it works on all websites.

**Comment by Gi4nt1** [comment]
I recently discovered PR and love it! Especially that I finally can watch YT without Google tracking me. The rest I don't use that much but still I was wondering if any of similar services exist for fb and ig? - Need it badly enough for work sometimes.

**Comment by opalvault** [comment]
I'd like to take a moment to put some eyes back on this issue, as I've come from https://github.com/humanetech-community/awesome-humane-tech and was disappointed to see redirects to old.reddit.com. It seems counter intuitive that an app focused on redirection to privacy respecting alternatives to common social media sites would then give options to redirect users to the very sites it's trying to avoid.

I've created a pull request (#323) to address at least some of @tio-trom's (in my eyes) valid points.

**Comment by SimonBrazell** [comment]
Hello @opalvault, yes I will address this in the near future, as I agree with @tio-trom's points here, I'll refrain from accepting PRs with new redirects in future without more scrutiny of the services they redirect to.

**Comment by opalvault** [comment]
Looking forward to future additions. Thank you for your work on this.
